ext/pcntl/tests/pcntl_signal.phpt

```diff
@@ -42,6 +42,6 @@ bool(false)
 Warning: pcntl_signal(): Invalid signal %s
 bool(false)
 
-Warning: pcntl_signal(): not callable is not a callable function name error in %s
+Warning: pcntl_signal(): Specified handler not callable is not callable in %s
```

Comment:
While this is an improvement to the message, I don't think it really addresses bug #78402, and this example here also reads quite weirdly. I'd suggest to use `"%s"` at least, so that it reads `Specified handler "not callable" is not callable` and `Specified handler "" is not callable` for the null case.

Reply:
Updated.
Also I've found the words "function name error" was actually meaningful, so I'd resurrected one.